src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/AutoRerouteV2Spec.groovy

```diff
@@ -563,7 +563,7 @@ class AutoRerouteV2Spec extends HealthCheckSpecification {
                 }
                 sleep(500)
             }
-            assert northboundV2.getFlow(firstFlow.flowId).statusInfo =~ /ISL (.*) become INACTIVE because of FAIL TIMEOUT (.*)/
+            assert northboundV2.getFlow(firstFlow.flowId).statusInfo =~ /ISL (.*) become INACTIVE(.*)/
```

Comment:
nit: Is it really related to the issue?

Reply:
yes... this is fix from @rtretyak. Without this fix, this test fails.

Reply:
Ok